components/inputs/test/input-time.test.js

```diff
@@ -0,0 +1,130 @@
+import '../input-time.js';
+import { aTimeout, expect, fixture, oneEvent } from '@open-wc/testing';
+
+const basicFixture = '<d2l-input-time label="label text"></d2l-input-time>';
+
+describe('d2l-input-time', () => {
+
+	let elem;
+
+	beforeEach(async() => {
+		elem = await fixture(basicFixture);
+	});
+
+	function getInput() {
+		return elem.shadowRoot.querySelector('.d2l-input');
+	}
+
+	describe('accessibility', () => {
+
+		it('passes all axe tests', () => {
+			expect(elem).to.be.accessible();
+		});
+
+		it('passes all axe tests when label is hidden', () => {
+			elem.setAttribute('label-hidden', 'label-hidden');
+			expect(elem).to.be.accessible();
+		});
+
+		it('passes all axe tests when disabled', () => {
+			elem.setAttribute('disabled', 'disabled');
+			expect(elem).to.be.accessible();
+		});
+
+		it('passes all axe tests when focused', async() => {
+			setTimeout(() => getInput(elem).focus());
+			await oneEvent(elem, 'focus');
+		});
+	});
+
+	describe('attribute reflection', () => {
+		it('should reflect disabled property to attribute', async() => {
+			elem.setAttribute('disabled', 'disabled');
+			await elem.updateComplete;
+			expect(elem.hasAttribute('disabled')).to.be.true;
+		});
+	});
+
+	describe('labelling', () => {
+
+		function getLabel() {
+			return elem.shadowRoot.querySelector('.d2l-input-label');
+		}
+
+		it('should display visible label', async() => {
+			expect(getLabel().innerText).to.equal('label text');
```

Comment:
Is this captured in visual-diff already?

Reply:
There are specific tests for `basic` and for `label-hidden`, but the visual-diff doesn't do anything to make sure they're different or that the label exists afaik. I mean going forward now that I've generated the goldens with them it should, but that doesn't seem like the intended purpose of visual-diff 🤔 I may be wrong.